src/content/profile-data.js

```diff
@@ -697,7 +722,7 @@ export function invertCallstack(thread: Thread): Thread {
 
 export function getSampleIndexClosestToTime(samples: SamplesTable, time: number): IndexIntoSamplesTable {
   // TODO: This should really use bisect. samples.time is sorted.
-  for (let i = 0; i < samples.length; i++) {
+  for (let i: number = 0; i < samples.length; i++) {
```

Comment:
is it really necessary ? I think this doesn't bring value here, and makes the code less readable.